Clients/src/presentation/components/Modals/ComplianceFeedback/ComplianceFeedback.tsx

```
@@ -0,0 +1,73 @@
+import CloudUpload from "../../../assets/icons/cloudUpload.svg";
+import RichTextEditor from "../../../components/RichTextEditor/index"; 
+
+interface AuditorFeedbackProps {
+  activeSection: string;
+}
+
+const AuditorFeedback: React.FC<AuditorFeedbackProps> = ({ activeSection }) => {
+  const [file, setFile] = useState<File | null>(null);
+
+  const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)